apiv2/src/admin/core/sejours/phase1/desistement/Desistement.service.ts

```diff
@@ -64,6 +66,9 @@ export class DesistementService {
             await this.affectationService.syncPlacesDisponiblesLignesDeBus(lignesDeBus);
         }
 
+        const sejours = await this.sejourGateway.findBySessionId(jeunes[0].sessionId!);
```

Comment:
```suggestion
        const sejours = await this.sejourGateway.findBySessionId(sessionId);
```
il est préférable de passer le sessionId en paramètre de la fonction, pour éviter au maximum les `[0]`.

Reply:
ok

Reply:
oubli pas de marquer à "resolved" quand tu l'as pris en compte